recipes/xspec-models-lite/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "xspec-models-lite" %}
+{% set version = "6.22" %}
```

From the URL it looks like the version is 6.22.1?
```suggestion
{% set version = "6.22.1" %}
```